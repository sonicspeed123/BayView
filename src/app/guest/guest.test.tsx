import { render } from '@testing-library/react';
import { describe, expect, it } from 'vitest';
import Page from './page';

describe('Guest Component', () => {
  it('matches the snapshot', () => {
    const { asFragment } = render(<Page />);
    expect(asFragment()).toMatchSnapshot();
  });
});
