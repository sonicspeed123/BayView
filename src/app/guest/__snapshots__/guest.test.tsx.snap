// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Guest Component > matches the snapshot 1`] = `
<DocumentFragment>
  <div
    class="guest"
  />
</DocumentFragment>
`;
